ui/src/types.ts

```diff
@@ -66,6 +66,21 @@ export interface TableFormData {
   sourceId: ResourceId,
 }
 
+export type DataTypeParam = 'format' | 'default' | 'language'
```

Comment:
Language is not a datatype param.

And on second inspection turns out that neither is `default` 😞

Reply:
It's not in CSVW, but it is in the UI (just because it makes sense)